test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -27,4 +27,12 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches.Count | Should Be 1
         $result.CompletionMatches[0].CompletionText | Should Be "pscustomobject"
     }
+    It "Issue#1345 - 'Import-Module -n<tab>' should work" {
```

Comment:
Is it best practice to place a issue number in `It` title? Maybe make a comment below?